package.json

```diff
@@ -32,8 +33,9 @@
     "zod": "^3.23.8"
   },
   "devDependencies": {
+    "@stylistic/eslint-plugin-ts": "^2.11.0",
```

Comment:
Why?

Reply:
Well DevOps said that there was a problem with `typescript-eslint/eslint-plugin` package. After I've installed the latest version I've found out that some rules are deprecated/moved to new packages. E.g. [member-delimiter-style](https://typescript-eslint.io/rules/member-delimiter-style/) rule.

Reply:
There is no longer a problem with packages in CI. Can we revert extra changes?

Reply:
Let's follow newer versions and get away from deprecated functionality